### ISSUE TITLE
Double bond stereo created for undefined bond when pasting SMILES

### ISSUE THREAD

**Issue by mwojcikowski** [epam/ketcher#2249]
**Steps to Reproduce**
1. Paste `FC=CF` to Ketcher
2. Save to Daylight SMILES

**Actual behavior**
Ketcher generates `F/C=C/F`

**Expected behavior**
Ketcher should generate same smiles `FC=CF`

**Screenshots**
![image](https://user-images.githubusercontent.com/2325645/220307399-cd3468c8-d931-48bc-b9dd-8741f5605f38.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version 109

**Ketcher version** [e.g. v2.4.2]. 
2.7.1 and 2.8.0-rc3 (demo instances)

**Comment by AlexanderSavelyev** [comment]
this issue happens because when you paste SMILES with unknown cis-trans information, Ketcher generates layout with specified cis-trans (coordinates can't be generated without cis-trans by definition). 
Question: should Ketcher just have a flag to skip cis-trans while saving into SMILES or only if input was with unknown cis-trans?

**Comment by mwojcikowski** [comment]
I do not think this should go under the flag. I consider this to be a bug and the output is wrong. 
The double bond should be marked as unknown and the double bond should be crossed or one of the single bonds marked as undefined

When using `FC=CF`in Marvin it's marked (stereo flag = 4) on single bond

```

  Mrv2114 06192311512D          

  4  3  0  0  0  0            999 V2000
   -3.0252    2.9436    0.0000 F   0  0  0  0  0  0  0  0  0  0  0  0
   -2.6127    3.6580    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
   -1.7877    3.6580    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
   -1.3752    4.3725    0.0000 F   0  0  0  0  0  0  0  0  0  0  0  0
  2  1  1  0  0  0  0
  3  2  2  0  0  0  0
  3  4  1  **4**  0  0  0
M  END
```

In RDKit it's marked as flag = 3 on double
```
     RDKit          2D
  4  3  0  0  0  0  0  0  0  0999 V2000
   -1.2990    0.7500    0.0000 F   0  0  0  0  0  0  0  0  0  0  0  0
    0.0000    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    1.2990    0.7500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    2.5981   -0.0000    0.0000 F   0  0  0  0  0  0  0  0  0  0  0  0
  1  2  1  0
  2  3  2  **3**
  3  4  1  0
M  END
